server_extensions_api/src/main/java/com/generalbytes/batm/server/extensions/IIdentitySimple.java

```diff
@@ -0,0 +1,155 @@
+     *
+     * @return external ID
+     */
+    String getExternalId();
+
+    /**
+     * Identity state, one of the following:
+     * <ul>
+     *      <li>{@link IIdentity#STATE_NOT_REGISTERED} = 0 - "Not Registered"</li>
+     *      <li>{@link IIdentity#STATE_REGISTERED} = 1 - "Registered"</li>
+     *      <li>{@link IIdentity#STATE_TO_BE_REGISTERED} = 2 - "Awaiting Registration"</li>
+     *      <li>{@link IIdentity#STATE_PROHIBITED} = 3 - "Rejected"</li>
+     *      <li>{@link IIdentity#STATE_ANONYMOUS} = 4 - "Anonymous"</li>
+     *      <li>{@link IIdentity#STATE_PROHIBITED_TO_BE_REGISTERED} = 5 - "Rejected Wants To Be Registered"</li>
+     *      <li>{@link IIdentity#STATE_TO_BE_VERIFIED} = 6 - "Awaiting Verification"</li>
+     * </ul>
+     *
+     * @return int value representing current identity state.
+     */
+    int getState();
+
+    /**
+     * Identity type, one of the following:
+     * <ul>
+     *     <li>{@link IIdentity#TYPE_INTERNAL} = 0 - must not be instantiated by extension</li>
+     *     <li>{@link IIdentity#TYPE_EXTERNAL} = 1 -this identity was created externally (by extension)</li>
+     * </ul>
+     *
+     * @return int value representing type, if not provided defaults to TYPE_INTERNAL
+     */
+    int getType();
+
+    /**
+     * Date when identity was created.
+     *
+     * @return {@link Date}
+     */
+    Date getCreated();
+
+    /**
+     * Date when identity state changed to STATE_REGISTERED
+     *
+     * @return {@link Date}
+     */
+    Date getRegistered();
+
+    /**
+     * Returns serial number of terminal that created this identity or was assigned to manually created one.
+     *
+     * @return serial number
+     */
+    String getCreatedByTerminalSerialNumber();
+
+    /**
+     * Returns identification of person who registered this identity.
+     *
+     * @return {@link IPerson} when known, {@code null} otherwise
+     */
+    IPerson getRegisteredBy();
+
+    /**
+     * Discount From Buy Profit Fee.
+     *
+     * @return Value of the discount, may return {@code null} when not set.
+     */
+    BigDecimal getVipBuyDiscount();
+
+    /**
+     * Discount From Sell Profit Fee.
+     *
+     * @return Value of the discount, may return {@code null} when not set.
+     */
+    BigDecimal getVipSellDiscount();
+
+    /**
+     * Last modification date.
+     *
+     * @return {@link Date}
+     */
+    Date getLastUpdatedAt();
+
+    /**
+     * Last date when identity was checked against watchlist(s).
+     *
+     * @return {@link Date}
+     */
+    Date getWatchListLastScanAt();
+
+    /**
+     * Informs if identity was banned due to presence in a watchlist.
+     *
+     * @return {@code true} if banned, {@code false} otherwise
+     */
+    boolean isWatchListBanned();
+
+    /**
+     * Deprecated, use {@link IIdentity#getNotes()} instead.
+     *
+     * @return Single note related to identity
+     */
+    @Deprecated
+    String getNote();
```

Comment:
Maybe remove - customers using this "new" interface won't need to have an option to use deprecated method